karma.coverage.conf.js

```diff
@@ -1,37 +1,31 @@
 // Karma configuration
+
 (function () {
     'use strict';
 
     module.exports = function (config) {
-        var sharedConfig = require('./karma.shared.js');
-
-        sharedConfig.files = sharedConfig.files.concat([
-            // test assets
-            'test-it/karma/utilities/ExtendJasmineTimeout.js',
-
-            // test specs
-            'test-it/karma/helloworld/*.Test.js',
-            'test-it/karma/publicapi/*.Test.js',
-            'test-it/karma/vtrsuite/*.Test.js',
-            'test-it/karma/http/*.Test.js',
-            'test-it/karma/javascriptinvoke/*.Test.js'
-        ]);
-
-        // level of logging
-        // possible values: config.LOG_DISABLE || config.LOG_ERROR || config.LOG_WARN || config.LOG_INFO || config.LOG_DEBUG
-        sharedConfig.logLevel = config.LOG_INFO;
+        var getSharedConfig = require('./karma.shared.js');
+        var sharedConfig = getSharedConfig(config);
+
+        sharedConfig.files = [
+            ...sharedConfig.filesPolyfills,
+            ...sharedConfig.filesInfrastructure,
+            'dist/wasm32-unknown-emscripten/release/vireo.js',
+            ...sharedConfig.filesSource,
+            ...sharedConfig.filesFixtures,
+            ...sharedConfig.filesSpecs
+        ];
 
         sharedConfig.browsers = [
-            'Firefox'
+            'FirefoxHeadless'
         ];
 
         sharedConfig.reporters = [
             'dots',
             'coverage'
         ];
 
-        sharedConfig.preprocessors['source/core/*.js'] = ['coverage'];
-        sharedConfig.preprocessors['source/io/*.js'] = ['coverage'];
+        sharedConfig.preprocessors['dist/wasm32-unknown-emscripten/release/vireo.js'] = ['coverage'];
```

Comment:
Does this mean we're generating a code coverage report for the karma tests?

Reply:
Only if you run the `npm run test-coverage` command